tests/test_pixel_breath.py

```diff
@@ -197,7 +197,7 @@ def _mock(*_args, **_kwargs) -> np.ndarray:
             name="Tidal impedance variation",
             unit=None,
             category="impedance difference",
-            time=np.linspace(0, 100),
+            time=np.linspace(0, 100, 100),
```

Comment:
This is also a lot of repeated code. Only the attributes `time` and `values` differ. Please put the creation of those within `if`/`else` blocks, and only create a SparseData object once.